src/js/components/RegisterPage/SignUp.js

```diff
@@ -1,139 +1,82 @@
 import React, { Component } from "react";
-import FormGroup from "react-bootstrap/FormGroup";
-import Row from "react-bootstrap/Row";
-import Form from "react-bootstrap/Form";
-import Button from "react-bootstrap/Button";
-import FormLabel from "react-bootstrap/FormLabel";
-import "./SignUp.sass";
+import Navbar from "../Navbar/Navbar";
+import "./SignUp.scss";
 
-export default class Signup extends Component {
-  constructor(props) {
-    super(props);
+class SignUp extends Component {
+    render()
+    {
+        let loc_navBarTitle = "MEMBER REGISTRATION";
+        let loc_navbarItems = [true, true, true, true];
 
-    this.state = {
-      isLoading: false,
-      email: "",
-      password: "",
-      confirmPassword: "",
-      confirmationCode: "",
-      newUser: null
-    };
-  }
+        return (
+            <div className="SignUp">
+                <Navbar titleFromParent={loc_navBarTitle} navbarItems={loc_navbarItems}/>
+                <div className="container ">
+                    <div className="row marginTop20px ">
+                            <h3 className="marginLeft20px">Register</h3>
+                    </div>
+                    <div className="row ">
+                        <div className="col-12 marginTop20px">
+                            <label >Name</label>
+                            <span><input className="formInput" type="text" value="jack"/></span>
```

Comment:
Why are you giving defaults values to your input?

Reply:
For display purposes, and for fast testing as that is faster than having to continuously re-enter information

Reply:
You can use an extension called Form Filler on your Chrome browser.